# Goal:
geometry_msgs/PoseStamped start_pose
geometry_msgs/PoseStamped end_pose
string tool_frame
bool reset_controllers
# default: True
float32 velocity
# default: 0.03
float32 err_pos_max
# default: 0.1
float32 err_ang_max
# default: 1.0
uint32 setup_steps
# default: 3
---
# Result:
string result
---
# Feedback message:
